app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -23,7 +23,7 @@ import '../request_handling/request_handler.dart';
 import '../service/buildbucket.dart';
 
 /// List of repos that require CQ+1 label.
-const Set<String> kNeedsCQLabelList = <String>{'flutter/flutter'};
+const Set<String> kNeedsCQLabelList = <String>{};
```

Comment:
We won't need the label anymore, would you mind removing it?

Reply:
label removed, and the corresponding logic/tests.